### PR TITLE
Fix problems with a hallucination's Initialize

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by Cyberboss** [comment]
Can someone tell me what the problem is? I don't see it

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -262,51 +262,57 @@ Gunshots/explosions/opening doors/less rare audio (done)
...
+	fakerune = image('icons/effects/96x96.dmi', landing_image_turf, "landing", layer = ABOVE_OPEN_TURF_LAYER)
+	fakebroken.override = TRUE
+	if(target.client)
+		target.client.images |= fakebroken
+		target.client.images |= fakerune
+	target.playsound_local(wall,'sound/effects/meteorimpact.ogg', 150, 1)
+	bubblegum = new(wall, target)
+	addtimer(CALLBACK(src, .proc/bubble_attack), 10)
+
+/obj/effect/hallucination/oh_yeah/proc/bubble_attack
```
Comment (AnturK) [inline code/modules/flufftext/Hallucination.dm]: ()

File: code/modules/flufftext/Hallucination.dm [hunk 2]
```diff
@@ -262,51 +262,57 @@ Gunshots/explosions/opening doors/less rare audio (done)
...
+	var/charged = FALSE //only get hit once
+	while(get_turf(bubblegum) != landing && target && target.stat != DEAD)
+		bubblegum.forceMove(get_step_towards(bubblegum, landing))
+		bubblegum.setDir(get_dir(bubblegum, landing))
+		target.playsound_local(get_turf(bubblegum), 'sound/effects/meteorimpact.ogg', 150, 1)
+		shake_camera(target, 2, 1)
+		if(bubblegum.Adjacent(target) && !charged)
+			charged = TRUE
+			target.Weaken(4)
+			target.staminaloss += 40
```
Comment (ChangelingRain) [inline code/modules/flufftext/Hallucination.dm]: augh! this should be adjustStaminaLoss(40)

**Comment by lzimann** [comment]
@Cyberboss

**Pull request merged**